### PR TITLE
Add reference to the >> and | operators for pipelines.

### PR DESCRIPTION
**Pull request by mariapython**
This comes after getting questions on what those operators represent.

### CODE REVIEW

**Comment by mariapython** [comment]
R: @aaltay

File: sdks/python/README.md [hunk 1]
```diff
@@ -163,8 +163,9 @@ The following examples demonstrate some basic, fundamental concepts for using Ap
 
 A basic pipeline will take as input an iterable, apply the
 beam.Create `PTransform`, and produce a `PCollection` that can
-be written to a file or modified by further `PTransform`s. The
-pipe operator allows to chain `PTransform`s.
+be written to a file or modified by further `PTransform`s.
+The `>>` operator is used to label PTransforms and
```
Comment (aaltay) [inline sdks/python/README.md]: Use ticks around PTransform, (like the line above) so that it renders correctly (e.g. \`PTransform\`)

Comment (mariapython) [inline sdks/python/README.md]: Thanks. Done.

**Comment by aaltay** [comment]
LGTM

**Comment by mariapython** [comment]
@robertwb: Please merge

**Pull request closed without merging**